### ISSUE TITLE
Quelle méthode adopter pour estimer le besoin en aide alimentaire ? 

### ISSUE THREAD

**Issue by martindaniel4** [dataforgoodfr/croixrouge#4]
Nous devons pouvoir estimer le besoin en aide alimentaire d'une zone géographique (ville, canton, département) et le comparer avec l'actuelle distribution proposée par les centres croix rouge.

Pour chaque centre croix rouge (avec nom de ville et département), nous disposons d'une vue agrégée sur les bénéficiaires, avec notamment : 
- Les tranches de revenu
- La CSP 
- Le nombre de membres de la famille 

La première approche à laquelle nous pensons est la suivante: consturire un proxy du besoin en aide alimentaire à partir de données ouvertes (taux de chômage, revenus par foyer, densité urbaine, toute autre données INSEE ou autre de niveau locale). Ce proxy serait construit/appris à partir des données de distribution actuelles. Les "outliers" d'un tel modèle mettraient en évidence les zones surinvesties / sous-investies.
- Cette approches est-elle convenable ? Quelles sont ses faibless ? 
- Quelle autre approche serait possible?

**Comment by RomainWarlop** [comment]
@martindaniel4 t'entends quoi par construire un proxy ? 

Niveau méthodo on peut construire un clustering de type EM à partir de l'ensemble des données que tu cites + taille de la ville, aide reçu, répartition des revenues (pour voir si fort écart dans la ville), proximiter à une grande ville, ... Ensuite on essaie à posteriori de ranker les clusters du plus nécessiteux au moins nécessiteux et on regarde le taux d'implantation de la croix rouge dans chaque cluster. T'en penses quoi ?

**Comment by martindaniel4** [comment]
@RomainWarlop : ça me parait top. Faut tester. On a aussi des données sur les 
bénéficiaires - ce qu ils demandent ainsi que certaines caractéristiques (revenu, sexe, charges etc..), je me dis qu on peut lier ça à des donnés INSEE. 

@samronsin, @c-o vous en pensez quoi ?

**Comment by RomainWarlop** [comment]
@martindaniel4 ok, du coup dans l'idéal il faut avoir toutes les variables dispo sur les bénéficiaires dans notre table servant au clustering, ça nous aidera pour le ranking a posteriori

**Comment by samronsin** [comment]
@martindaniel4 Je pense aussi que des données INSEE pourraient être très utiles pour modéliser la demande. Un des problèmes avec les données CRF c'est qu'on ne verra pas de demande là où ils ne sont pas...

**Comment by samronsin** [comment]
@RomainWarlop @martindaniel4 Une fois qu'on a modélisé une demande, on ne manquera pas de méthodes pour la comparer avec l'offre CRF (peut-être le plus simple à modéliser par ailleurs). Je dis juste ça pour ne pas s'arrêter au clustering, qui serait une excellente approche parmi d'autres, a priori.

Du coup ma suggestion serait de commencer par l'offre CRF telle qu'elle est actuellement histoire de se faire la main, puis la demande avec des données INSEE, puis plancher sur des offres CRF optimisées. Qu'en pensez-vous?

**Comment by martindaniel4** [comment]
@samronsin @RomainWarlop :+1:

**Comment by RomainWarlop** [comment]
@samronsin, @martindaniel4 yes je suis aussi d'accord pour les données INSEE. Pour moi y'a deux tables distinctes, une avec toute sorte de données dont INSEE et une avec les données sur les bénéficiaires. La première sert pour le clustering, la deuxième pour le ranking des clusters. Y'a pas besoin de géo loc avec la deuxième table, elle sert au descriptif des profils. Je vais essayer de lister les variables accessibles.

@samronsin yes on peut commencer par là.

**Comment by martindaniel4** [comment]
@RomainWarlop @samronsin j'ai ajouté des données INSEE [là](https://github.com/dataforgoodfr/croixrouge/tree/master/data/INSEE), c'est inspiré de [ce post](http://ants.builders/blog/18-02-2014/predicting-abstention-rate-using-open-data.html). Je vais les documenter dans le Wiki

@RomainWarlop si tu commences à documenter des tables, hésite pas à renseigner ça sur le [Wiki ici](https://github.com/dataforgoodfr/croixrouge/wiki/Description-des-tables)

**Comment by martindaniel4** [comment]
@samronsin @RomainWarlop y a aussi les autres centres de distribution à prendre en compte (Restos du coeur, secours populaire, banque alimentaire)

**Comment by samronsin** [comment]
J'ai demandé à un pote à Étalab s'ils avaient ces données, on verra ce qu'il en dit...

**Comment by martindaniel4** [comment]
@samronsin cool. J'ai aussi demandé à un contact aux Restos du coeur.

**Comment by c-o** [comment]
@samronsin "Un des problèmes avec les données CRF c'est qu'on ne verra pas de demande là où ils ne sont pas...", c'était justement pour ça l'histoire du proxy: si on arrive à corréler un modèle CRF avec un modèle INSEE, on peut extrapoler pour les zones sans CRF
@RomainWarlop j'aime bien l'idée de clustering car il pourrait donner des informations intelligibles si on arrive à tagger correctement les clusters (genre cluster des familles nombreuse en zone non dense avec 1 travail temps partiel). Pour que la CRF arrive localement à mieux prouver leur nécessité, c'est plus "vendeur" d'avoir des règles intelligibles.
pour moi CRF ça voulait dire Conditionnal Random Field, mais ça ne marchait pas dans la conversation :)

**Comment by martindaniel4** [comment]
@c-o, @RomainWarlop, @samronsin Je vous conseille la lecture du [rapport 2014 de l'action sociale de la croix rouge](https://github.com/dataforgoodfr/croixrouge/blob/master/misc/crf_bilandas2014.pdf). 

Il y a notamment une carte, du taux de pauvreté monétaire et des `quartiers prioritaires` définis par la Croix Rouge, cf : 

> L’identification de ces nouveaux quartiers se fonde sur le critère unique de la pauvreté, c’est-à-dire la concentration des populations ayant des ressources inférieures à 60 % du revenu médian.

![capture d ecran 2015-04-15 a 10 28 08](https://cloud.githubusercontent.com/assets/695006/7154880/94ec0e06-e35a-11e4-9d41-dfcf4a1aaf1a.png)

Il semble également qu'on puisse récupérer la localisation des bénévoles, puisqu'à la page 9, il y a une carte de la répartition des 33 000 bénévoles : 

![capture d ecran 2015-04-15 a 10 37 40](https://cloud.githubusercontent.com/assets/695006/7155012/8f45b4b0-e35b-11e4-87f5-8a393aad1b60.png)

La partie sur l'aide alimentaire et vestimentaire se trouve p.37. Intéressant notamment l'exemple des jardiniers solidaires. C'est exactement ce genre de partenariat, local, avec la grande distribution qu'il faudrait arriver à monter

> Les jardiniers solidaires
> L’Unité Locale de Moulins a choisi de sensibiliser les jardiniers à les offrir à la Croix Rouge afin d’améliorer la diversité des denrées proposées lors des aides alimentaires en renforçant la part de légumes et fruits frais. Et pourquoi ne pas inciter également les cultivateurs à mettre quelques plants destinés à notre association ? Cette initiative, relayée par TF1 et FR3 en 2013, a permis d’accroître considérablement les apports de l’aide alimentaire en récupérant 1,5 T de légumes et fruits en 2013, chiffre largement dépassé en 2014.

**Comment by martindaniel4** [comment]
@samronsin @RomainWarlop @lefko200 @rquilliet @c-o @NebJ : est ce que vous êtes chauds pour faire une rencontre ensemble (dans la vraie vie) et taffer 2h ensemble sur un premier livrable ? 

On pourrait se donner rendez vous lundi ou mardi semaine prochaine. 

Who's in ?

**Comment by rquilliet** [comment]
Yo Martin,

Pas possible pour moi lundi / mardi. Suis à l'étranger ...

Mais complètement opé pour la vraie vie :)

Remi

**Comment by RomainWarlop** [comment]
Yo, ouais trop, la vraie vie c'est cool. 
Perso le jeudi me va un peu mieux la semaine prochaine, lundi pas possible et mardi s'il le faut j'viendrai.

Romain

**Comment by samronsin** [comment]
@martindaniel4 I'm in!
